dockerfiles/hpcc/values.schema.json

```diff
@@ -299,6 +299,15 @@
           "description": "The number of slave pods",
           "minimum": 1
         },
+        "lingerPeriod": {
+          "type": "integer",
+          "description": "[Optional] Seconds to keep Thor instance running, waiting for more graphs to execute",
+          "minimum": 1
```

Comment:
would it be better to allow 0 in order to disable it?

Reply:
it is disabled by default, if the property is not present.
So this is saying if used/present, assert that it is >=1